docker-compose.dev.yml

```diff
@@ -1,13 +1,11 @@
 x-autogenerated: This file is autogenerated
-version: '3'
```

Comment:
dropping version, as it was deprecated already [docs](https://docs.docker.com/compose/compose-file/#compose-file)